src/NServiceBus.Transport.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -117,7 +116,11 @@ public Task ChangeConcurrency(PushRuntimeSettings limitations, CancellationToken
         {
             maxConcurrency = limitations.MaxConcurrency;
             Logger.InfoFormat("Calling a change concurrency and reconnecting with new value {0}.", limitations.MaxConcurrency);
-            _ = Task.Run(() => Reconnect(messageProcessingCancellationTokenSource.Token), cancellationToken);
+            _ = Task.Run(async () =>
```

Comment:
I know you are not changing this offloading, and I'm aware of the current flaw with the change concurrency being done in the middle of the pipeline. As far as I recall, though, all other transport implementations execute the start stop sequence inline. Should we either align all other or do it here the same?

Reply:
yeah I forgot to commit that change - my bad.